RAMP/unpacker.py

```diff
@@ -59,8 +62,16 @@ def _validate_zip_file(zip_file, max_bundle_size_mb):
     exceed a certain threshold.
     """
     infolist = zip_file.infolist()
-    # We're expecting exactly two files.
-    if len(infolist) != 2:
+    # We're expecting to have .so file, module.json and deps dir.
+    files_counter = 0
+
+    for file in infolist:
+        if file.filename.endswith(".so") or file.filename == "module.json":
+            files_counter += 1
```

Comment:
the whole file counting with +1 and -1 feels not intuitive at all.
simply do 
```
so_files = len([file for file in infolist if file.filename.endswith(".so")])
json_files = len([file for file in infolist if file.filename == "module.json"]) 

if len(infolist) > 3:
    raise...

if so_files != 1:
    raise...

if json_files != 1:
    raise...
```
and you can do the same for deps if you want